src/apps/loan-list/utils/helpers.ts

```diff
@@ -83,4 +83,24 @@ export const getStackedItems = (
   return returnLoans;
 };
 
+export const getLoanDeliveryDate = (
+  loanType: LoanType,
+  formatDate: (date: string) => string,
```

Comment:
I really bugs me that we are passing date strings around when we should be using Date objects. But that is a matter that does not need to be solved in this PR